### PR TITLE
feat: initial `addProject` test with WebdriverIO

### PR DESCRIPTION
**Pull request by ndom91**
## ☕️ Reasoning

- Add initial "Add Project" E2E test
- Sister PR to #4661

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,6 +9,11 @@ generated-do-not-edit/
 .sentryclirc
 .DS_Store
 
+# E2E Artifacts
+**/e2e/videos
+apps/desktop/one-vbranch-on-integration
+apps/desktop/remote
```
Comment (mtsgrd) [inline .gitignore]: These need updating I think?

File: apps/desktop/e2e/scripts/init-repositories.sh [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+(
+  cd remote
+
+  git config user.email "test@example.com"
+  git config user.name "Test User"
+  git config init.defaultBranch master
+
+  echo first >file
+  git add . && git commit -m "init"
+)
```
Comment (mtsgrd) [inline apps/desktop/e2e/scripts/init-repositories.sh:37]: What's the purpose of the `(...)`?

File: apps/desktop/wdio.conf.ts [hunk 3]
```diff
@@ -9,11 +8,12 @@ let tauriDriver: ChildProcess;
 export const config: Options.WebdriverIO = {
 	hostname: '127.0.0.1',
 	port: 4444,
-	specs: ['./e2e/**/*.spec.js'],
+	specs: ['./e2e/tests/**/*.spec.js', './e2e/tests/**/*.spec.ts'],
```
Comment (mtsgrd) [inline apps/desktop/wdio.conf.ts]: We can probably remove the .js one right?

File: packages/ui/src/lib/inputs/Button.svelte [hunk 4]
```diff
@@ -27,6 +27,7 @@
 		icon?: keyof typeof iconsJson | undefined;
 		help?: string;
 		helpShowDelay?: number;
+		'data-testid'?: string;
```
Comment (mtsgrd) [inline packages/ui/src/lib/inputs/Button.svelte]: Can we camelCase this please?

File: apps/desktop/e2e/scripts/init-repositories.sh (hunk 2 above)
Comment (ndom91) [reply apps/desktop/e2e/scripts/init-repositories.sh:37]: This runs those commands in a subprocess, so we dont have to `cd` back out of each repository directory, for example

File: packages/ui/src/lib/inputs/Button.svelte (hunk 4 above)
Comment (ndom91) [reply packages/ui/src/lib/inputs/Button.svelte]: Yup good call :+1:

**Review by mtsgrd — APPROVED** [review]

**Pull request merged**